src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -199,4 +199,17 @@ public Metadata()
                     storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ProvideDateAndTimeCompletions"));
         }
     }
+
+    internal static class CompletionOptionsExtensions
+    {
+        public static bool ShouldShowItemsFromUnimportNamspaces(this CompletionOptions options)
```

Comment:
>ShouldShowItemsFromUnimportNamspaces [](http://example.com/codeflow?start=27&length=36)

Does not need to be an extension method. You can move it directly to CompletionOptions

Reply:
done